test/tests/functional/pbs_mom_dynamic_resource_ext.py

```diff
@@ -60,6 +60,36 @@ def create_mom_resources(self, resc_name,
                                         validate=True)
         self.scheduler.add_resource(resc_name)
 
+        return fp
+
+    def create_mult_mom_resources(self, resc_name,
+                                  resc_type, resc_flag,
+                                  script_body):
+        """
+        helper function for create multiple mom_resources.
```

Comment:
helper function for creating

Reply:
Also can we combine these 2 functions related to adding mom resources. Just a suggestion but if it adds too much complexity then this is also okay.

Reply:
combining 2 functions is complicated, because for function  "create_mult_mom_resources" list are  passed as a parameter and for function "create_mom_resources" only values are passed as parameter. if we want to combine 2 functions then for single value also we need to create list and then that list pass to the function as a parameter.